war/hs_err_pid3655.log

```diff
@@ -0,0 +1,692 @@
+#
```

Comment:
That was an accident. I did not change gitignore settings so I don't know why that was commited. Now I removed it. I think may be I will make a new pull requst later with one commit.